scripts/bench_python_cpp_algos.py

```diff
@@ -83,6 +91,7 @@ def run_sssp(property_graph: PropertyGraph, input_args, source_node_file):
 
         for source in sources:
             with time_block(f"sssp on {source}"):
+                source = source.strip("\n")
```

Comment:
I'm not sure what's going on here. source shouldn't be a string here.

Reply:
Ah, yes. I have separate version of this script and it was string. My bad! Thank you for pointing this out!